### ISSUE TITLE
Generate audio avatar responses from elevenlabs

### ISSUE THREAD

**Issue by benbot** [Oneirocom/Magick#1032]
AS a magick user
WHEN I receive a response from the avatar
THEN that response is converted to audio via elevenlabs and played back to me

**Comment by benbot** [comment]
@ultrabot2022 is having some issues getting responses from the agent in order to send to elevanlabs

**Comment by ultrabot2022** [comment]
@benbot fixed the issue of getting responses from the agent. I can work with elevanlabs for audio with text

**Comment by benbot** [comment]
blocked on some secrets handling work. We need a way to share elevelabs secrets without relying on node context.

**Comment by fire** [comment]
Superseded by: https://github.com/Oneirocom/Magick/issues/1077?
